app/src/main/java/com/duckduckgo/app/fire/fireproofwebsite/ui/FireproofWebsitesViewModel.kt

```diff
@@ -69,6 +72,12 @@ class FireproofWebsitesViewModel(
         )
     }
 
+    fun loadFavicon(url: String, view: ImageView) {
+        viewModelScope.launch {
```

Comment:
the `viewModelScope` outlives the `ImageView` passed in here which makes me nervous. if you rotated the screen while the favicon was loading, is there a risk that we'd be trying to load an image into an `ImageView` which should be dead (because it belonged to the previous, unrotated `Activity`) does that matter?

Reply:
I've changed that so it uses the Activity rather than the ViewModel. This involves quite a few changes, i.e. passing the lifecycle owner to the adapter as well as the faviconManager but it will avoid this problem.